stitches.config.ts

```diff
@@ -0,0 +1,70 @@
+import { createStitches } from '@stitches/react';
+
+export const { styled, css, getCssText } = createStitches({
+    theme: {
+        media: {
+            bp1: '(min-width: 640px)',
```

Comment:
Why you didn't use the media for responsive support?

Reply:
I wanted to get the running app out and demoed and get some feedback before diving into fixing these details